wrangler.toml

```diff
@@ -0,0 +1,3 @@
+name = "bot-tg"
+main = "src/index.ts"
+compatibility_date = "2023-03-30"
```

Comment:
Are these configs/event var?
Maybe add this file to `.gitignore` and then make a `wrangler.toml-EXAMPLE` without the values

Reply:
Not really, they instruct wrangler to deploy onto right place, not related to runtime thing.
Anyway created `wrangler.toml-EXAMPLE` and instruction to update.